container.go

```diff
@@ -1866,3 +1867,43 @@ func (c *Container) SetRunningConfigItem(key string, value string) error {
 	}
 	return nil
 }
+
+// ConsoleLog allows to perform operations on the container's in-memory console
+// buffer.
+func (c *Container) ConsoleLog(opt ConsoleLogOptions) ([]byte, error) {
+	c.mu.Lock()
+	defer c.mu.Unlock()
+
+	lxcConsoleLog := C.struct_lxc_console_log{
```

Comment:
I know I'm just nitpicking but lxcConsoleLog is a long variable name :) How about cl?

Reply:
Sure, I'll update. :)

Reply:
@caglar10ur updated.